lib/tmx.lua

```diff
@@ -391,7 +391,7 @@ end
 local load_fixtures = function(world, layer, layer_idx, entity_spawn_callback)
   local fixtures = {}
   for _, object in ipairs(layer.objects) do
-    if object.name and object.type == 'entity' then
+    if object.name then
```

Comment:
This removes the need to set the type to "entity" for every entity created in Tiled. If you give an object a name it is assumed to be an entity and otherwise it is a boundary wall. Having other object types besides entities and boundary walls could be particularly useful in more complex maps (read: in other games) but no need to account for something we're not using until we need it.